sdks/jwt/icon/f63ac48f-ac5d-474c-bb39-e0010ddd99f9.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "sdks/jwt/Icon",
```

Comment:
As this is an SDK for the "jsonwebtoken" npm package, I'd change the namespace to "sdks/jsonwebtoken"